src/components/utils.js

```diff
@@ -10,11 +10,18 @@ function bindFunctions(obj, functionNames) {
     });
 }
 
-function pick(obj, names) {
+
+export function pick(obj, names) {
     if (typeof names === 'string') {
         names = names.split(/\s+/);
     }
     return _.pick(obj, names);
 }
 
-export { bindFunctions, pick };
\ No newline at end of file
+
+export function incrementMonth([year, month], by) {
```

Comment:
While I don't have any indication that this won't work, I'm always wary of any roll-your-own date handling code. Are there no other libraries in JS that could do this?

Reply:
This date is just a pair of numbers, year and month, that the user selects. I don't judge it worth introducing the complication of introducing a date-arithmetic package like Moment or DateMath, and converting this pair back and forth from their representations, just to accomplish "add *n* months". I'll put in tests, because you're right that it could be wrong.

Reply:
Blast, now you have me wondering if it *is* worth the complication of using full date representations.